### DISCUSSION TITLE
Something went wrong while executing your query on security vulnerability

### DISCUSSION THREAD

**Discussion by runely** [community/community#129451] [Mobile]
Not able to swipe a security vulnerability notification as read. It shows this red error

![image](https://github.com/community/community/assets/25814673/7afc5ba7-1f69-46de-81d7-04bca6f28f90)

<details>
<summary>ℹ️ Additional Debug Info</summary>
GitHub for iOS, Version 1.165.0 (133064262)
Device: iPhone 14 Pro Max (iOS 17.6)
</details>

**Comment by eliperkins** [comment]
Ah, thanks for this feedback @runely. The request ID you posted is quite helpful in helping us identify this.

I'll create an issue on the Security Vulnerability product team's repo to get this resolved, since we can see just where the error is coming from. I'll follow up with you once it's addressed!

Thanks for helping us build a high-quality product in GitHub Mobile 💪
